### ISSUE TITLE
since RN 25: import React from 'react-native' is deprecated

### ISSUE THREAD

**Issue by rkostrab** [bodyflex/react-native-simple-modal#3]
Please replace these lines:

```
import React from 'react-native';
const {
   View,
   Component,
   StyleSheet,
   TouchableOpacity,
   Animated,
   PropTypes,
   Platform,
   BackAndroid
} = React;
```

to

```
import React, { Component,PropTypes } from 'react';
import {
   View,
   StyleSheet,
   TouchableOpacity,
   Animated,
   Platform,
   BackAndroid
} from 'react-native;
```

Here are RN notes: https://github.com/facebook/react-native/releases/tag/v0.25.1

Thanks! :)

**Comment by jpgarcia** [comment]
just sent a PR with the fix which also closes #4 , hopefully @bodyflex will publish soon a new version :)

**Comment by httnn** [comment]
this is fixed now in v3.0.4 thanks to @jpgarcia!
